src/gateway_api/models/types/response/state/entity/details/metadata/collection.rs

```diff
@@ -0,0 +1,8 @@
+use crate::prelude::*;
+
+#[derive(
+    Deserialize, Serialize, Clone, PartialEq, Eq, Debug, uniffi::Record,
```

Comment:
Yes, since we are exposing `StateEntityDetailsResponse` (which has a `Vec<StateEntityDetailsResponseItem>`, and each item has an `EntityMetadataCollection`).

Reply:
.... ok, there is really no reason for `StateEntityDetailsResponse` to be UniFFI exposed, but since it is already, so be it for now.